BitwardenShared/UI/Vault/VaultItem/ViewItem/ViewLoginItem/Extensions/CipherView+Update.swift

```diff
@@ -118,6 +118,23 @@ extension CipherView {
         )
     }
 
+    /// Creates an `SSHKeyItemState` representation of the cipher.
+    ///
+    /// This function converts the `sshKey` information of the cipher into an `SSHKeyItemState`,
+    /// which is used to manage and display SSH key data in the UI.
+    ///
+    /// - Returns: An `SSHKeyItemState` representing the SSH key information of the cipher.
+    ///
+    func sshKeyItemState() -> SSHKeyItemState {
+        // TODO: PM-10401 create state when SDK is updated
+        SSHKeyItemState(
+            isPrivateKeyVisible: false,
+            privateKey: "TestPrivateKey",
```

Comment:
🤔 Should these be "Test" or just empty?

Reply:
The user can't trigger this flow so I'm fine with any content. Maybe "Test" or "TODO" make more sense to explicitly indicate something is needed to be changed there yet.